build/shade/EnvironmentParameters.shade

```diff
@@ -0,0 +1,20 @@
+use namespace='System'
+ 
+functions @{
+    static string GetEnvironmentParameter(string name)
+    {
+        return GetEnvironmentParameter(name, s => s);
+    } 
+
+    static T GetEnvironmentParameter<T>(string name, Func<string, T> valueFn)
+    {
+        var value = valueFn(Environment.GetEnvironmentVariable(name));
+        Console.WriteLine(name + ": " + (value != null ? value.ToString() : "(null)"));
+        return value;
+    }
+
+    static Func<string, string> DefaultTo(string defaultValue)
```

Comment:
I'll make the change. I wanted to keep it all within the `GetEnvironmentParameter` functionality but indeed it's a pain to debug Sake scripts.